src/asynciterable.ts

```diff
@@ -101,6 +101,7 @@ export abstract class AsyncIterableX<T> implements AsyncIterable<T> {
     thisArg?: any
   ): AsyncIterableX<TResult> {
     const fn = bindCallback(selector, thisArg, 2);
+    /* tslint:disable */
```

Comment:
this is cause two classes are referenced in circular, no way to satisfy lint by reordering classes.